decidim-core/lib/decidim/participatory_space_manifest.rb

```diff
@@ -50,5 +50,9 @@ def seed!
     def route_name
       super || model_class_name.demodulize.underscore
     end
+
+    def public_spaces(&block)
+      @public_resources ||= block
```

Comment:
Shouldn't this be `@public_spaces` instead? Or, following my suggestion: `@participatory_spaces`.